cask2formula

```diff
@@ -74,7 +74,7 @@ class CaskTransform < Parslet::Transform
         left + " " + right
     }
     rule(:font   => simple(:font)) {
-      "(share/\"fonts\").install #{replace_cask_version_properties(font.sub(/"(.*\/)/, '"../\1'))}"
+      "(share/\"fonts\").install #{replace_cask_version_properties(font.sub(/"((?:(?!(otf|ttf)\/)[[:blank:][:punct:]]?.)+\/)/i, '"../\1'))}"
```

Comment:
Any character. Actually I don't remember why I wrote like that and what is different than
```"((?:(?!(otf|ttf)\/).*)+\/)``` Maybe, first I wrote something
```"((?:(?!(otf|ttf))[[:blank:][:punct:]]?.)+\/)``` and changed it little bit and forgot about...
You are totally right

Forget it for now, I will open a new PR because :drum: I found the solution. I hope so... I suppose it will be easier.
I'm sorry for taking your time with this request